extensions/tokenizers/src/main/java/ai/djl/huggingface/tokenizers/HuggingFaceTokenizer.java

```diff
@@ -61,13 +62,15 @@ public final class HuggingFaceTokenizer extends NativeResource<Long> implements
     private boolean cleanupTokenizationSpaces;
     private boolean stripAccents;
     private boolean addPrefixSpace;
+    private final Map<String, String> options;
```

Comment:
We don't need keep options as a member variable, we can pass it down to `applyConfig()`